.github/l10n/check_l10n_issues.py

```diff
@@ -8,11 +8,121 @@
+            content = json.load(file)
+
+            if not content:
+                print(f"No content found in: {filepath}")
+                sys.exit(1)
+            return content
+    except FileNotFoundError:
+        print(f"File not found: {filepath}")
+        sys.exit(1)
+    except Exception as e:
+        print(f"An error occurred while loading {filepath}: {e}")
+        sys.exit(1)
+
+# Check if it uses shared strings. If so, pull the value for “title”, “subtitle”, then look within "blocks"
+def getSharedStringsInManifest(mainifest_contents):
+    shared_string_ids = []
+    # If it includes the key “message” and if within "message", there is a “usesSharedStrings” key that is true...
+    if "message" in manifest_contents and "usesSharedStrings" in manifest_contents["message"] and manifest_contents["message"]["usesSharedStrings"]:
+        #...collect all the string IDs
+        if "title" in manifest_contents["message"]:
+            shared_string_ids.append(manifest_contents["message"]["title"])
+        else:
+            print(f"No title found for: {addon}")
+            sys.exit(1)
+        if "subtitle" in manifest_contents["message"]:
+            # It is acceptable if there is no subtitle (but title is required)
+            shared_string_ids.append(manifest_contents["message"]["subtitle"])
+        if not "blocks" in manifest_contents["message"]:
+            print(f"No blocks found for: {addon}")
+            sys.exit(1)
+        for block in manifest_contents["message"]["blocks"]:
+            if not "content" in block:
+                print(f"No content found in {addon} for {block}")
+                sys.exit(1)
+            if isinstance(block["content"], str):
+                shared_string_ids.append(block["content"])
+            elif isinstance(block["content"], list):
+                # This is the contents of a bulleted or ordered list; go one layer deeper.
+                for list_item in block["content"]:
+                    if isinstance(list_item["content"], str):
+                        shared_string_ids.append(list_item["content"])
+                    else:
+                        print(f"No content found in {addon} for {list_item}")
+                        sys.exit(1)
+            else:
+                print(f"Content found in {addon} was unknown type for {block}")
+                sys.exit(1)
+    return shared_string_ids
+
+### 1. Check Addons' shared strings - all string IDs should be present in the translation file. ###
+# Find list of all addon folders
+addon_path = os.path.join(vpn_root_folder, "addons")
+try:
+    addon_list = [item for item in os.listdir(addon_path) if os.path.isdir(os.path.join(addon_path, item))]
+except FileNotFoundError:
+    print(f"Path not found: {addon_path}")
+    sys.exit(1)
+except Exception as e:
+    print(f"An error occurred when finding all addons: {e}")
+    sys.exit(1)
+
+if len(addon_list) == 0:
+    print(f"No themes found")
```

Comment:
Themes?

Reply:
Sorry, copy/paste error (from `check_colors.py`) - I've updated this